generators/python-v2/ast/src/TypeInstantiation.ts

```diff
@@ -200,6 +204,20 @@ export class TypeInstantiation extends AstNode {
                 assertNever(this.internalType);
         }
     }
+
+    private writeStringWithTripleQuotes({ writer, value }: { writer: Writer; value: string }): void {
+        writer.write('"""');
+        const parts = value.split("\n");
+        const head = parts[0] + "\n";
+        const tail = parts.slice(1).join("\n");
+        writer.write(this.escapeString(head));
+        writer.writeNoIndent(this.escapeString(tail));
```

Comment:
You'll notice that this `writeNoIndent` is used, but as the test below shows, it doesn't work as I'd expect.